src/app/activity/bounty/List.ts

```diff
@@ -46,30 +48,78 @@ export const listBounty = async (request: ListRequest): Promise<any> => {
+	return await sendMultipleMessages(listUser, dbRecords, request.guildId, channelName, IOUList);
 };
 
-const sendMultipleMessages = async (listUser: GuildMember, dbRecords: Cursor, guildId: string, bountyChannelName: string): Promise<any> => {
-	const listOfBounties = [];
-	while (listOfBounties.length < 10 && await dbRecords.hasNext()) {
-		const record: Bounty = await dbRecords.next();
-		const messageOptions: MessageEmbedOptions = await generateListEmbedMessage(record, record.status, guildId);
-		listOfBounties.push(messageOptions);
+const sendMultipleMessages = async (listUser: GuildMember, dbRecords: Cursor, guildId: string, bountyChannelName: string, IOUList: boolean): Promise<any> => {
+	if (IOUList) {
+		while (await dbRecords.hasNext()) {
+			const record: Bounty = await dbRecords.next();
+			const messageOptions: MessageEmbedOptions = await generateListEmbedMessage(record, record.status, guildId);
+			if (record.status == BountyStatus.open) {
+				messageOptions.footer = {
+					text: '💰 - paid | ❌ - delete ',
+				};
+			}
+			const message: Message = await (listUser.send( { embeds: [messageOptions] } ));
+			if (record.status == BountyStatus.open) {
+				await message.react('💰');
+				await message.react('❌');
+			}
+		}
+	} else {
+		const listOfBounties = [];
+		while (listOfBounties.length < 10 && await dbRecords.hasNext()) {
+			const record: Bounty = await dbRecords.next();
+			const messageOptions: MessageEmbedOptions = await generateListEmbedMessage(record, record.status, guildId);
+			listOfBounties.push(messageOptions);
+		}
+		await (listUser.send({ embeds: listOfBounties }));
+		await listUser.send({ content: `Please go to ${bountyChannelName} or your DMs to take action.` });
 	}
-	await (listUser.send({ embeds: listOfBounties }));
-	return await listUser.send({ content: `Please go to ${bountyChannelName} or your DMs to take action.` });
 };
 
 export const generateListEmbedMessage = async (bountyRecord: Bounty, newStatus: string, guildID: string): Promise<MessageEmbedOptions> => {
+	let fields = [];
+	if (bountyRecord.isIOU) {
+		fields = [
+			{ name: 'Bounty Id', value: bountyRecord._id.toHexString(), inline: false },
+			{ name: 'Reward', value: bountyRecord.reward.amount + ' ' + bountyRecord.reward.currency.toUpperCase(), inline: true },
+			{ name: 'Status', value: newStatus, inline: true },
+		]
+
+	} else {	
+		fields = [
+			{ name: 'Bounty Id', value: bountyRecord._id.toHexString(), inline: false },
+			{ name: 'Criteria', value: bountyRecord.criteria, inline: false },
+			{ name: 'Reward', value: bountyRecord.reward.amount + ' ' + bountyRecord.reward.currency.toUpperCase(), inline: true },
+			{ name: 'Status', value: newStatus, inline: true },
+			{ name: 'Deadline', value: formatDisplayDate(bountyRecord.dueAt), inline: true },
+			{ name: 'Created by', value: bountyRecord.createdBy.discordHandle, inline: true },
+		]
+	}
+
+	if (bountyRecord.resolutionNote) {
```

Comment:
I like that this shows up in list - just have to be transparent with users that their resolutionNotes are viewable by anyone who runs list

Reply:
Added "Optional public notes" to the command help